.github/config/out_of_tree_extensions.cmake

```
@@ -118,7 +118,6 @@ duckdb_extension_load(sqlite_scanner
 duckdb_extension_load(sqlsmith
         GIT_URL https://github.com/duckdb/duckdb_sqlsmith
         GIT_TAG 721460ff1f31ce1dc1e4a9c4a55c0faf0b466dcb
```

```suggestion
        GIT_TAG 9729cade201987b22ec7fc4b700f6d34e1e4111f
```
Forgot, also this needs to be bumped to AFTER absorbing the patches.